### DISCUSSION TITLE
Can i use async functions within the helpers?

### DISCUSSION THREAD

**Discussion by kratoskat** [pkosiec/mongo-seeding#200] [Q&A]
Trying to do some api calls in one of the helper functions before the data is imported but it doesnt seem to work. 

Can you confirm whether i am able to run async functions within the helpers?

**Comment by pkosiec** [comment]
Hi @kratoskat,
What do you mean by helpers? 
Do you mean transformers? If so, then unfortunately not: https://github.com/pkosiec/mongo-seeding/blob/main/core/src/transformer/transformer.ts#L20-L22. But you can easily iterate over collections and do your own stuff before calling `import`.

Hope that helps!

**Reply by kratoskat** [reply]
Actually just realised that Transformers are a bit different. I'm talking about what you call "helpers" in your docs such as this https://github.com/pkosiec/mongo-seeding/tree/main/examples/import-data/example/helpers.

**Reply by pkosiec** [reply]
Oh, I get it now. Of course, you can use async/await there. These helpers are JS/TS files, so it's up to you how they are written 🙂

**Comment by kratoskat** [comment]
it does, thanks :)
